plugin/Record/RecordManagerReadListViewEstate.php

```diff
@@ -219,16 +233,20 @@ public function getRowByName($listviewName, $type = null)
 	 *
 	 */
 
-	private function readPicturetypesByListviewId($listviewId)
+	public function getPicturetypesByListviewId(int $listviewId) : array
 	{
 		$prefix = $this->getTablePrefix();
 		$pWpDb = $this->getWpdb();
 
-		$sqlPictures = "SELECT `picturetype`
-				FROM {$prefix}oo_plugin_picturetypes
-				WHERE `listview_id` = ".esc_sql($listviewId);
+		$sql = $pWpDb->prepare(
+			"SELECT *
```

Comment:
Sollte hier nicht auch nur `picturetype ` selektiert werden?
```suggestion
			"SELECT `picturetype`
```